MahMaterialDragablzMashUp/PaletteSelectorViewModel.cs

```diff
@@ -1,38 +1,42 @@
-﻿using MaterialDesignColors;
+﻿using System;
+using MaterialDesignColors;
 using MaterialDesignThemes.Wpf;
 using System.Collections.Generic;
 using System.Windows.Input;
 
 namespace MahMaterialDragablzMashUp
 {
-    public class PaletteSelectorViewModel
+    public class PaletteSelectorViewModel : PaletteHelper
```

Comment:
We don't need inheritance here.  If anything we should be illustrating how people can inject in the PaletteHelper, previously we just instantiated it on demand, but if people wanted to TDD a view model to manage palettes they'd most likley inject a mocked version into the constructor.  So maybe just create the PaletteHelper in the constructor.